xml/System.Windows.Xps.Packaging/IXpsFixedDocumentWriter.xml

```diff
@@ -1,241 +1,246 @@
+      <MemberSignature Language="ILAsm" Value=".property instance class System.Printing.PrintTicket PrintTicket" />
+      <MemberSignature Language="DocId" Value="P:System.Windows.Xps.Packaging.IXpsFixedDocumentWriter.PrintTicket" />
+      <MemberSignature Language="VB.NET" Value="Public Property PrintTicket As PrintTicket" />
+      <MemberSignature Language="F#" Value="member this.PrintTicket : System.Printing.PrintTicket" Usage="System.Windows.Xps.Packaging.IXpsFixedDocumentWriter.PrintTicket" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; property System::Printing::PrintTicket ^ PrintTicket {  void set(System::Printing::PrintTicket ^ value); };" />
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>ReachFramework</AssemblyName>
+        <AssemblyVersion>3.0.0.0</AssemblyVersion>
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Printing.PrintTicket</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Sets default print options for the <see cref="T:System.Windows.Documents.FixedDocument" />.</summary>
+        <value>The value must be a valid PrintTicket instance.
+             Note:  The PrintTicket can only be assigned to prior to if being committed to the package.  The commit happens when a valid PrintTicket is set and a subsequent flush on the document occurs.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ Setting this property does not validate or modify the specified <xref:System.Printing.PrintTicket> for a particular <xref:System.Printing.PrintQueue>.  If needed, use the <xref:System.Printing.PrintQueue.MergeAndValidatePrintTicket%2A> method to create a <xref:System.Printing.PrintQueue>-specific <xref:System.Printing.PrintTicket> that is valid for a given printer.  
+  
+   
+  
+## Examples  
+ The following example shows how to use the <xref:System.Windows.Xps.Packaging.IXpsFixedDocumentWriter.PrintTicket%2A> property to associate a <xref:System.Printing.PrintTicket> with a <xref:System.Windows.Documents.FixedDocument>.  
+  
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_Wpf/XpsCreate/CPP/XpsCreate.cpp" id="Snippetxpscreateaddpkgcontent":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_Wpf/XpsCreate/CSharp/XpsCreate.cs" id="Snippetxpscreateaddpkgcontent":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_Wpf/XpsCreate/visualbasic/xpscreate.vb" id="Snippetxpscreateaddpkgcontent":::  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">PrintTicket has already been committed.</exception>

```

Comment:
<exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">The print ticket has already been committed.</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Not a valid <see cref="T:System.Printing.PrintTicket"/> instance.</exception>